### PR TITLE
Improve logging for mobile development

### PR DESCRIPTION
**Pull request by ademar111190**
Fixes https://github.com/breez/c-breez/issues/681

- Increase log level of app to info
- Filter fine logs from logcat/xcode

### CODE REVIEW

File: lib/logger.dart [hunk 1]
```diff
@@ -30,8 +30,10 @@ class BreezLogger {
     if (kDebugMode) {
       Logger.root.onRecord.listen((record) {
         // Dart analyzer doesn't understand that here we are in debug mode so we have to use kDebugMode again
-        if (kDebugMode) {
-          print(_recordToString(record));
+        if (record.level >= Level.INFO) {
```
Comment (roeierez) [inline lib/logger.dart:33]: I think the sdk signer and node logs have debug level. We do need to include them by default.

**Review by roeierez — CHANGES_REQUESTED** [review]
Looks good. Only a small comment regarding log verbosity.

**Review by erdemyerebasmaz — COMMENTED** [review]
If our aim is to get rid of "Fine" level SDK logs, aka all SDK logs except ERROR, WARN, INFO, DEBUG level logs. My suggestion is not logging anything on default case on the `registerBreezSdkLog()`, or set `Logger.root.level` to `Level.CONFIG;` on BreezLogger constructor with these changes that convert certain app logs from fine to info.

File: lib/logger.dart (hunk 1 above)
Comment (ademar111190) [reply lib/logger.dart:33]: This filter is only applied for printing (the log file will keep all logs); Do you want to print these to log too @roeierez ?

Comment (roeierez) [reply lib/logger.dart:33]: Is there a reason to differentiate?

Comment (ademar111190) [reply lib/logger.dart:33]: My rational is the bug report as I mention here: https://github.com/breez/c-breez/issues/681#issuecomment-1766553792

I'll remove from the file; in future we can add back if we want.

@erdemyerebasmaz I'll keep the naming changes to do not lose it; different from the sdk logs, the app logs are very few and add a lot of information about user behavior, essential on bug reports

**Review by roeierez — APPROVED** [review]

File: lib/logger.dart [hunk 2]
```diff
@@ -26,7 +26,7 @@ void shareLog() async {
 
 class BreezLogger {
   BreezLogger() {
-    Logger.root.level = Level.ALL;
+    Logger.root.level = Level.INFO;
```
Comment (erdemyerebasmaz) [inline lib/logger.dart:29]: @ademar111190 Think we want to log anything equal to and above CONFIG, not INFO here.

**Pull request merged**